ibm_catalog.json

```diff
@@ -69,32 +61,14 @@
                   "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-secrets-manager-private-cert-engine/main/reference-architecture/secrets_manager_private_cert_engine.svg",
                   "type": "image/svg+xml"
                 },
-                "description": "This architcture supports creating secrets manager private certificates engine within a secrets manager instance."
+                "description": "This architecture supports creating a Secrets Manager private certificates engine within a Secrets Manager instance. The private certificates engine enables you to issue and manage internal TLS/SSL certificates without relying on external certificate authorities. It supports creating and managing root and intermediate certificate authorities, defining certificate templates, and automating certificate issuance for internal applications and services."
```

Comment:
Please use the same name across everywhere. (Notice the uppercase)
`Secrets Manager private certificates engine` -> `Secrets Manager Private Certificates Engine`